core/consensus/babe/primitives/src/digest.rs

```diff
@@ -52,17 +52,13 @@ pub enum BabePreDigest {
 		authority_index: super::AuthorityIndex,
 		/// Slot number
 		slot_number: SlotNumber,
-		/// Chain weight (measured in number of Primary blocks)
-		weight: BabeBlockWeight,
```

Comment:
Where should this go then?

Reply:
it'll be maintained off-chain; it's a bit redundant to put it in the block header.